torch/fx/experimental/symbolic_shapes.py

```diff
@@ -1351,12 +1369,13 @@ def repr_symbol_to_source():
             f"not in {repr_symbol_to_source()}.  If this assert is failing, it could be "
             "due to the issue described in https://github.com/pytorch/pytorch/pull/90665"
         )
-        return self.source_ref(self.symbol_to_source[expr][0])
+        source = self.symbol_to_source[expr][0]
+        return self.source_ref(source, self.sources)
```

Comment:
You don't have to pass sources in here anymore, just have source_ref close over it

Reply:
I don't understand?